config/default.yml

```diff
@@ -2343,6 +2343,9 @@ Lint/TopLevelReturnWithArgument:
   Description: 'Detects top level return statements with argument.'
   Enabled: true
   VersionAdded: '0.89'
+  VersionChanged: '<<next>>'
```

Comment:
I think it is unnecessary because this is a bugfix and doesn't imply a default configuration changes.

```suggestion
```

Reply:
I thought that `VersionChanged` should always be changed when there is any change in the default configuration of the Cop, regardless of the purpose or background of the change.  Shouldn't it be changed in the case of a bug fix?

- https://github.com/rubocop/rubocop/pull/6293#issuecomment-423640987

Reply:
As a short answer yes for here. it's unnecessary given the current context I think (It's not a one-size-fits-all) .

Reply:
I see, thank you for explaining 👍